src/components/swapv2/TokenInfoV2.tsx

```diff
@@ -84,8 +84,8 @@ const TokenInfoV2 = ({
     chainId,
   })
 
-  const showToken1 = !isEmptyData(tokenInfo1)
-  const showToken2 = !isEmptyData(tokenInfo2)
+  const showToken1 = !isEmptyData(tokenInfo1) && isHardCode
+  const showToken2 = !isEmptyData(tokenInfo2) && isHardCode
```

Comment:
I personally dont like this variable name. Perhaps we should try something like `isSupported`, `isV1`, ...